index.yaml

```diff
@@ -9,6 +9,17 @@ indexes:
 # automatically uploaded to the admin console when you next deploy
 # your application using appcfg.py.
 
+- kind: ApiAuthAccess
+  properties:
+  - name: event_list
+  - name: owner
+  - name: expiration
+
+- kind: ApiAuthAccess
```

Comment:
Are both of these necessary? I don't understand indexes fully, but I would assume the above includes this one. To test, you can delete both indexes, restart the dev server, and then run through both ApiAuthAcces.query() calls.